src/App.js

```diff
@@ -1,16 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+const chatLogData = chatMessages;
```

Comment:
I'm only seeing this variable referenced once, we should be able to remove it and use `chatMessages` on line 9 where we set up the initial `updatedChatLogData` state.